redwood-treehouse-host/src/commonMain/kotlin/app/cash/redwood/treehouse/TreehouseAppContent.kt

```diff
@@ -84,7 +149,7 @@ internal class RealBinding<A : AppService>(
   private var treehouseUiOrNull: ZiplineTreehouseUi? = null
 
   /** Only accessed on [TreehouseDispatchers.ui]. */
-  private var firstDiff = true
+  private var diffCount = 0
```

Comment:
Why this change?

Reply:
Yeah, this is off topic from the rest of the PR. I added `codeLoadCount` to replace tracking the previous binding, and I thought these two should be symmetric.

I’m contemplating about changing `CodeListener` to something less opinionated.

```
fun onCodeLoading(content: Content, view: TreehouseView, count: Int)
fun beforeDiffApplied(content: Content, view: TreehouseView, count: Int)
```

We don’t need this flexibility yet.